pkg/api/handlers/containers_stats.go

```diff
@@ -52,7 +57,7 @@ func (s *APIServer) statsContainer(w http.ResponseWriter, r *http.Request) {
 
 	stats, err := ctnr.GetContainerStats(&libpod.ContainerStats{})
 	if err != nil {
-		internalServerError(w, errors.Wrapf(err, "Failed to obtain container %s stats", name))
+		InternalServerError(w, errors.Wrapf(err, "Failed to obtain Container %s stats", name))
```

Comment:
Why did you capatilize this?

Reply:
In case these error functions are needed in the other package.